Userland/Applications/PixelPaint/Layer.h

```diff
@@ -56,18 +56,23 @@ class Layer
     DeprecatedString const& name() const { return m_name; }
     void set_name(DeprecatedString);
 
-    void flip(Gfx::Orientation orientation);
-    void rotate(Gfx::RotationDirection direction);
-    void crop(Gfx::IntRect const& rect);
-    void resize(Gfx::IntSize new_size, Gfx::Painter::ScalingMode scaling_mode);
-    void resize(Gfx::IntRect const& new_rect, Gfx::Painter::ScalingMode scaling_mode);
-    void resize(Gfx::IntSize new_size, Gfx::IntPoint new_location, Gfx::Painter::ScalingMode scaling_mode);
+    enum NotifyClients {
+        YES,
+        NO
+    };
```

Comment:
Nit-pick: Please call these `Yes` and `No` instead of them being in all capitals. :^)